src/dotnet/dotnet-migrate.ts

```diff
@@ -1,27 +1,16 @@
+import { filepaths } from "src/_common/generators";
+
 const completionSpec: Fig.Spec = {
   name: "migrate",
   args: {
     name: "project",
     description:
       "The project or solution file to operate on. If not specified, the command searches the current directory for one. If more than one solution or project is found, an error is thrown",
     isOptional: true,
-    generators: {
-      template: ["filepaths", "folders"],
-      filterTemplateSuggestions(param) {
-        const suffix = ".sln";
-        const fileNames = ["project.json", "global.json"];
-
-        return param.filter((file) => {
-          if (typeof file.name === "string") {
-            const fileName = file.name;
-
-            return fileName.endsWith(suffix) || fileNames.includes(fileName);
-          }
-
-          return false;
-        });
-      },
-    },
+    generators: filepaths({
+      extensions: ["sln"],
+      matches: /(project\.json)|(global\.json)/g,
```

Comment:
Should this use `equals` rather than `matches` ?

```suggestion
      equals: ["project.json", "global.json"]
```

Also potentially should also have `showFolders: "always"`

Reply:
I used a regex because it was using .includes js method before. Equals requires strict equality of the string while that regex allows characters before and after `project.json` and `global.json`. I'll check if those files can for e.g. have a prefix and update the pr accordingly.

Reply:
I think that was an error with the initial generator: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-migrate